### DISCUSSION TITLE
AppleALC kext

### DISCUSSION THREAD

**Discussion by JecerBenH** [stevedat/HP-Elitebook-830-G6-Hackintosh#6] [Q&A]
Hello Steve , could u please send me the applealc kext my email is Jasser.benhammouda@icloud.com and thank you

**Comment by stevedat** [comment]
AppleALC kext you can find on GitHub page, but HP 830 G6 like mine using correct layout ID yet no input audio neither internal micro nor jack 3.5 micro. You might buy a usb headphone to use with AppleALC kext. It works flawlessly for me on Monterey.

**Comment by stevedat** [comment]
iMess me on dat.tranphat@icloud.com, I can share my EFI if your laptop is same specs with mine
